content/operate/oss_and_stack/install/install-redis/install-redis-on-linux.md

```diff
@@ -14,7 +14,12 @@ Most major Linux distributions provide packages for Redis.
 
 ## Install on Ubuntu/Debian
 
+Add the repository to the APT index, update it, and install Redis:
```

Comment:
We may want to add an extra first step, just to make sure all dependencies we refer to are satisfied:

```
sudo apt-get install lsb-release curl gpg
```

Reply:
Done. Thanks again, @yossigo!